reconcile/cli.py

```diff
@@ -102,9 +103,21 @@
 from utils.unleash import get_feature_toggle_state
 
 
+def before_breadcrumb(breadcrumb):
```

Comment:
Can you add a docstring to explain what this does? I expect we will eventually want to add more things in there to mask other types of sensitive informaton